datasafe-storage/datasafe-storage-api/src/main/java/de/adorsys/datasafe/storage/api/SchemeDelegatingStorage.java

```diff
@@ -56,7 +56,7 @@ public OutputStream write(WithCallback<AbsoluteLocation, ? extends ResourceWrite
     }
 
     private StorageService service(AbsoluteLocation location) {
-        String protocol = location.location().asURI().getScheme();
+        String protocol = location.location().asURI().getScheme().split("-")[0];
```

Comment:
I don't think split is needed, there should be direct match for protocol

Reply:
done